staging/src/k8s.io/client-go/transport/spdy/spdy.go

```diff
@@ -77,11 +77,12 @@ func NewDialer(upgrader Upgrader, client *http.Client, method string, url *url.U
 	}
 }
 
-func (d *dialer) Dial(protocols ...string) (httpstream.Connection, string, error) {
+func (d *dialer) Dial(remotePorts []uint16, protocols ...string) (httpstream.Connection, string, error) {
 	req, err := http.NewRequest(d.method, d.url.String(), nil)
 	if err != nil {
 		return nil, "", fmt.Errorf("error creating request: %v", err)
 	}
+	req.Header.Set("port", fmt.Sprintf("%d", remotePorts))
```

Comment:
I don't think adding this data in headers is a good way to get this info into the audit log, since the headers aren't actually used or required to match the ports the connections are actually made to